src/resources/js/views/manager.js

```diff
@@ -200,11 +200,23 @@ tribe.events.views.manager = {};
 	 */
 	obj.onSubmit = function( event ) {
 		event.preventDefault();
-		var $container = $( this );
-		var formData = Qs.parse( $container.serialize() );
 
-		// pass the data to the request using `tribe-events-views`
-		obj.request( formData['tribe-events-views'], $container );
+		// The submit event is triggered on the form, not the container.
+		var $form = $( this );
+		var $container = $form.closest('.tribe-events');
+		var nonce = $container.data('view-rest-nonce');
+
+		var formData = Qs.parse($form.serialize());
+		var viewData = formData['tribe-events-views'] || {};
```

Comment:
why are we doing this relaying of information instead of allowing the form data to do its job?
This would make subviews more complicated, would love to see if we could keep it using the form fields, and if URL field is not available uses the `window.location.href`.

Reply:
I'm not sure I understand the question here.